### DISCUSSION TITLE
Add warning icons also to the node inspector

### DISCUSSION THREAD

**Discussion by miv391** [godotengine/godot-proposals#5531] [Editor]
Warning icons in the scene view are of course very useful. I propose adding those same warning icons and warning texts also to the node inspector. It would then be faster to find the issues that must be taken care of. I photoshopped "a shape must be provided" warning icon to the following image:

![kuva](https://user-images.githubusercontent.com/49998025/193469649-db43ef4f-4d28-47f2-b626-f70ce4f8f858.png)

**Comment by Mickeon** [comment]
I have a few ideas, but it would require slight refactoring of the way warnings are generated and detected. In addition to this, it would be nice to redirect the user to the core issue when they click on the warning.

**Comment by Calinou** [comment]
This requires implementing support for property-scoped node (and resource) configuration warnings, which is planned for a future 4.x release but not 4.0.
